src/content/operationDashboard/DashboardView.tsx

```diff
@@ -20,45 +21,42 @@ export type DashboardViewProps = {
   loading: boolean;
 };
 
-const StyledViewHeader = styled(Box)({
+const StyledPageContainer = styled(Box)({
   background: `url(${bannerSvg})`,
```

Comment:
The background in the Figma seems to be different. Are we going with the one you are currently using?

Reply:
TBD, but it doesn't seem like it. If I get an answer, I'll update it in the future.